tests/apps/courses/test_models_course_run.py

```diff
@@ -623,6 +623,8 @@ def test_models_course_run_mark_dirty_any_field(self):
         stub = CourseRunFactory(
             sync_mode="manual",
             catalog_visibility=CourseRunCatalogVisibility.COURSE_ONLY,
+            offer=CourseRunOffer.SUBSCRIPTION,
```

Comment:
@JoaoGarcao You have to use the setting `RICHIE_DEFAULT_COURSE_RUN_OFFER` that you have added to to the settings.py